selected_polygon.geojson

```diff
@@ -16,24 +16,24 @@
         "coordinates": [
           [
             [
-              1568800.0,
-              5199000.0
+              1569282.0,
+              5198773.0
             ],
             [
-              1577600.0,
-              5199000.0
+              1576867.0,
+              5198773.0
             ],
             [
-              1577600.0,
-              5192500.0
+              1576867.0,
+              5193101.0
             ],
             [
-              1568800.0,
-              5192500.0
+              1569282.0,
+              5193101.0
             ],
             [
-              1568800.0,
-              5199000.0
+              1569282.0,
+              5198773.0
             ]
```

Comment:
It is still the Kaiapoi area, I've put it back to the original catchment area we were previously using, not sure how it had been changed once.

Reply:
I made it bigger so that it was easier to preload the whole area and then draw rectangles within it, but let me know if that was causing issues

Reply:
I thought it was done by accident, so I've put it back so that we didn't need to re-generate the HydroDEM for the area. Do you want me to put it back to the bigger area?

Reply:
Yes please, I chose a bigger area because the old one was too small, hopefully it doesn't cause too much re-generation

Reply:
done